### PR TITLE
Update Docs re Global Keyboard Shortcuts

### PR DESCRIPTION
**Pull request by Kidglove57**
Update Documentation regarding Global Keyboard Shortcuts as these do not work in the Web app

### CODE REVIEW

File: docs/getting-started/tipstricks.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ If you ever make a mistake, pressing <Key mod="cmd" k="z" /> will undo, and pres
 
 ## Keyboard shortcuts
 
-A few global shortcuts:
+Firstly, a few global shortcuts for the Electron App that is still in development. Please be aware that these global shortcuts do not work in the web app:
 
 - <Key mod="cmd" k="1" /> will show the budget
 - <Key mod="cmd" k="2" /> will show reports
 - <Key mod="cmd" k="3" /> will show all accounts
-- <Key mod="cmd" k="o" /> will close the file and list other available files to open
```
Comment (j-f1) [inline docs/getting-started/tipstricks.md:17]: Are you planning to add this somewhere else?

Comment (Kidglove57) [reply docs/getting-started/tipstricks.md:17]: Now added. I was confusing myself thinking it meant CTRL 0 (Zero) not O!

File: docs/getting-started/tipstricks.md [hunk 2]
```diff
@@ -9,12 +9,16 @@ If you ever make a mistake, pressing <Key mod="cmd" k="z" /> will undo, and pres
...
+ 
+BUT here is one that WILL work in the web app as well:
+- <Key mod="cmd" k="o" /> (O not zero) this will close the file and list other available files to open
+
+Substitute "ctrl" for "cmd" if you are on Windows.
```
Comment (j-f1) [inline docs/getting-started/tipstricks.md:21]: This line isn’t needed — we automatically switch the shortcut displayed on the doc page to show Ctrl if we detect you’re not running on a Mac!

Comment (Kidglove57) [reply docs/getting-started/tipstricks.md:21]: That's clever - I had no idea! Will do.

**Review by j-f1 — APPROVED** [review]
Glad you’re getting the hang of it!

**Pull request merged**